apps/manager_app/manager_api.c

```diff
@@ -0,0 +1,131 @@
+/**
+ * @file    manager_api.c
+ * @author  Cypherock X1 Team
+ * @brief   Title of the file.
```

Comment:
Either remove this or add meaningful description

Reply:
added